platformio.ini

```diff
@@ -12,7 +12,7 @@
 [build_envs]
 default_envs =
 ; *** Uncomment by deleting ";" in the line(s) below to select version(s)
-;                tasmota
+                tasmota

```

Comment:
Platformio.ini file shouldn´t be modified

Reply:
Yes, this is right. Sorry, this is my 1st pull-request and i did not succeed in removing my platform.ini changes from pull-request. So i mentioned in the description to remove the platformio.ini changes.

Reply:
> I did not succeed in removing my platform.ini changes from pull-request.

you can:

1- go to https://github.com/arendst/Tasmota/blob/development/platformio.ini
2- click on the pen on the upper right
3- copy everything
4- go to your platformio.ini of your branch https://github.com/pharaocurt/Tasmota/blob/ds18x20-arithm-mean/platformio.ini
5- click on the pen on the upper right
6- paste everything
7- click on the green button to commit and you are done.